src/components/annual-report/section4/section4ChartGenerator.js

```diff
@@ -0,0 +1,101 @@
+import i18n from '@dhis2/d2-i18n'
+
+export const generateSection4Chart = (canvasId, chartInfo) => {
+    const dataPoints = chartInfo.values
+        .filter(({ invalid }) => !invalid)
+        .map(({ x, y, divergent, name }) => ({
+            x,
+            y,
+            marker: {
+                symbol: divergent ? 'diamond' : 'circle',
+            },
+            custom: {
+                name,
+                xLabel: chartInfo.x,
+                yLabel: chartInfo.y,
+            },
+        }))
+
+    const xMax = Math.max(...dataPoints.flatMap(({ x }) => x))
+    const xMaxThreshold = (xMax / 100) * chartInfo.threshold
+
+    return {
+        chart: {
+            renderTo: canvasId,
+            type: 'scatter',
+        },
+        series: [
+            {
+                name: i18n.t('Orgunits'),
```

Comment:
I know it's "Orgunits" in the old app, but I think we can do better 😁 
```suggestion
                name: i18n.t('Org units'),
```